src/notifications/signal.h

```diff
@@ -41,17 +41,19 @@ enum class SubscriberPosition { kFront, kBack };
 template <typename... Args> class Signal {
 public:
 	/** Wrapper around a callback function. */
-	struct SignalSubscriber {
-		explicit SignalSubscriber(const Signal& p, const std::function<void(Args...)>& c)
-		   : parent_(p), callback_(c) {
+	class SignalSubscriber {
+	public:
+		explicit SignalSubscriber(const Signal& p, const std::function<void(Args...)> c)
```

Comment:
isn't explicit meaningless for multi argument constructors?

Reply:
Not entirely meaningless: https://stackoverflow.com/a/39122237
but not that useful either. Removed it.